recurly/notifications/notifications_test.go

```diff
@@ -0,0 +1,178 @@
+package notifications_test
+
+import (
+	"encoding/xml"
+	"net/http"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/blacklightcms/go-recurly/recurly"
+	"github.com/blacklightcms/go-recurly/recurly/notifications"
+)
+
+func TestParse_SuccessfulPaymentNotification(t *testing.T) {
+	xmlFile, err := os.Open("xml/successful_payment_notification.xml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer xmlFile.Close()
+
+	req, err := http.NewRequest("POST", "/", xmlFile)
```

Comment:
`os.Open` returns `*os.File`, which [implements](https://golang.org/pkg/os/#File.Read) the `io.Reader` interface. You can pass `xmlFile` directly into `notifications.Parse` without the extra step of creating an `*http.Request`.